Changed all 'NED board' references to 'FRD body'. Also cleaned up mix…

rblilja opened the pull request:
…ing of m/s/s and m/s^2 to use the latter. There are still some messages that needs to be looked at for correct reference frame: 'vehicle_rates_setpoint.msg' and 'vehicle_attitude_setpoint.msg

Solves issue: #16328

dagar commented:
There are conflicts in quite a few files, can you rebase this on current master?

dagar commented on msg/sensor_correction.msg:
```diff
@@ -21,29 +21,29 @@ float32[3] gyro_scale_2		# gyro XYZ scale factors in the sensor frame
 # Note the corrections are in the sensor frame and must be applied before the sensor data is rotated into body frame
 # corrections for uORB instance 0
-float32 baro_offset_0		# barometric pressure offsets in the sensor frame in m/s/s
+float32 baro_offset_0		# barometric pressure offsets in the sensor frame in m/s^2
```
```suggestion
float32 baro_offset_0		# barometric pressure offsets in the sensor frame in Pascals.
```

rblilja commented:
> There are conflicts in quite a few files, can you rebase this on current master?

@dagar I added current master as upstream and rebased my fork. Then pushed my origin to my fork master. Does that suffice? Sorry - I'm new to rebasing.

dagar commented:
> > There are conflicts in quite a few files, can you rebase this on current master?
> 
> @dagar I added current master as upstream and rebased my fork. Then pushed my origin to my fork master. Does that suffice? Sorry - I'm new to rebasing.

Yes that would be fine, but it doesn't look like that branch made it to github as it still reports the conflicts.

<img width="477" alt="Screen Shot 2020-12-06 at 1 32 13 PM" src="https://user-images.githubusercontent.com/84712/101288990-77a12080-37c7-11eb-8e51-2fd4b48f807a.png">

Merging with latest PX4 master is also fine.

rblilja commented:
@dagar Got messy. Closing this one and making a fresh.
